### PR TITLE
🌱 Remove unused mixin from controller suite test template

### PR DESCRIPTION
**Pull request by Adirio**
# Description
Remove `RepositoryMixin` from controller suite tests.

# Motivation
Controller suite test do not use the repository field for anything, so there is no need to inject it into the template.

P.S.: I have checked for every template and these two are the only ones with an unnecessary mixin or field.

### CODE REVIEW

**Review by camilamacedo86 — APPROVED** [review]
/approve
/lgtm 

we can get it merged since it is a small cleanup

**Comment by camilamacedo86** [comment]
/hold

In this case, the title mojo should :ghost:

**Comment by camilamacedo86** [comment]
HI @Adirio, 

Maybe the 👻 was removed intentionally. It has an issue raised for a while: https://github.com/kubernetes-sigs/kubebuilder-release-tools/issues/9. Let's use 🌱 for this kind of scenario so far.

**Comment by Adirio** [comment]
This PR mis passing every test, but tide is not recognizing the Travis test success. @camilamacedo86 already tried to close and re-open it twice, with no luck. @DirectXMan12 @mengqiy is there any other thing we can do?

**Comment by camilamacedo86** [comment]
HI @Adirio, 

Could you please try to rebase it with the master and push? I believe that it will solve this scenario.

**Comment by Adirio** [comment]
Done, it requires the `lgtm`label again.

**Pull request merged**
